recipe/meta.yaml

```diff
@@ -1,18 +1,18 @@
-{% set version = "1.16.2" %}
+{% set version = "1.16.4" %}
 
 package:
   name: automake
   version: {{ version }}
 
 source:
   url: http://ftp.gnu.org/gnu/automake/automake-{{ version }}.tar.gz
-  sha256: b2f361094b410b4acbf4efba7337bdb786335ca09eb2518635a09fb7319ca5c1
+  sha256: 8a0f0be7aaae2efa3a68482af28e5872d8830b9813a6a932a2571eac63ca1794
   patches:
     - 0001-apple-silicon.patch
     - 0002-apple-silicon.patch
```

Comment:
Looks like these patches no longer apply

cc @isuruf

Reply:
Both of them can be dropped as they were merged upstream.

Reply:
Thanks! 😄 Done below